### PR TITLE
chore(deps): Included dependency review

### PR DESCRIPTION
**Pull request by naveensrinivasan**
> Dependency Review GitHub Action in your repository to enforce dependency
> reviews on your pull requests.
> The action scans for vulnerable versions of dependencies introduced by package version
> changes in pull requests,
> and warns you about the associated security vulnerabilities.
> This gives you better visibility of what's changing in a pull request,
> and helps prevent vulnerabilities being added to your repository.

https://docs.github.com/en/code-security/supply-chain-security/understanding-your-software-supply-chain/about-dependency-review#dependency-review-enforcement
Signed-off-by: naveen <172697+naveensrinivasan@users.noreply.github.com>

### CODE REVIEW

File: .github/workflows/depsreview.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v1
```
Comment (rejas) [inline .github/workflows/depsreview.yaml:14]: could you update it to v2 ?

**Review by rejas — CHANGES_REQUESTED** [review]
sorry for having this PR ignored for so long @naveensrinivasan

File: .github/workflows/depsreview.yaml (hunk 1 above)
Comment (khassel) [reply .github/workflows/depsreview.yaml]: did the update and prettier, could be merged @rejas (we can add changelog later with another PR)

Comment (rejas) [reply .github/workflows/depsreview.yaml]: so you can change the PR but not add an entry?

Comment (khassel) [reply .github/workflows/depsreview.yaml]: did this in the online editor, could edit the file but did not find so far how to add another file ...

Comment (rejas) [reply .github/workflows/depsreview.yaml]: Maybe a quick PR with changelog entry and some dependency updates :-) ?

Comment (rejas) [reply .github/workflows/depsreview.yaml]: https://github.com/MichMich/MagicMirror/pull/2960 :-)

**Pull request merged**